_includes/header.html

```diff
@@ -78,7 +78,16 @@
             <li><a href="{{ site.baseurl }}/project/logos/">Logos and design</a></li>
             <li><a href="{{ site.baseurl }}/project/public-meetings/">Public Meetings</a></li>
 			<li><a href="{{ site.baseurl }}/project/team/">Team</a></li>
-            <li><a href="http://apache.org/licenses/LICENSE-2.0.html">License</a></li>
+          </ul>
+        </li>
+        <li class="dropdown">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><img src="http://www.apache.org/foundation/press/kit/feather_small.png" alt="Apache Logo" style="height:24px;">Apache Software Foundation<span class="caret"></span></a>
```

Comment:
the img src should begin with `://`, not `http://` -- that will use HTTPS if the current page is HTTPS, and HTTP otherwise. As-is, good browsers will (I believe) warn that insecure resources are included.